src/ccnl-core/src/ccnl-prefix.c

```diff
@@ -165,6 +184,12 @@ ccnl_prefix_appendCmp(struct ccnl_prefix_s *prefix, unsigned char *cmp,
 
 // TODO: This function should probably be moved to another file to indicate that it should only be used by application level programs
 // and not in the ccnl core. Chunknumbers for NDNTLV are only a convention and there no specification on the packet encoding level.
+/**
+ * Add a chunk number to a prefix
+ * @param prefix prefix on which the chunknum should be added to (return pointer)
```

Comment:
In general I would add the @param[in], @param[out] options so you easily see if a (pointer) parameter changes after it has been passed to a function